### DISCUSSION TITLE
Could I deploy poinz to our interna servers

### DISCUSSION THREAD

**Discussion by ices9527** [Zuehlke/poinz#344] [General]
I and team are fans of  this poinZ app. However there are some restriction on the internal network, which make the access not stable. Could I build and deploy the app to  our on premised server? Is that allowed or any limits?

**Comment by xeronimus** [comment]
Hello @ices9527 ,

I would be interested in learning what kind of restrictions do have an impact on Poinz. Is there anything we can do to make Poinz work better in your setting?

You asked about deploying locally, and whether there are limitations.  If you mean legal limitations, there are none. Poinz is [MIT Licensed](https://en.wikipedia.org/wiki/MIT_License)   (see [/LICENSE](https://github.com/Zuehlke/poinz/blob/master/LICENSE) ).
So you are allowed to build Poinz and deploy it wherever you want :)

**Reply by ices9527** [reply]
Glad to hear your quick feedback. Actually the netwrok denies the access to https://www.poinz.app/ site, rather than the app itself, as so I try to get a local deployment in the intranet. :) It really helps.
